docs/manual/manual_redundant_throttle.rst

```diff
@@ -0,0 +1,113 @@
+##############################
+Redundant Throttle Support
+##############################
+
+***************************
+About Redundant Throttle
+***************************
+
+Vertiq's redundant throttle support is a safety feature that allows your module to accept multiple throttle sources' commands simultaneously, and to switch between them should 

```

Comment:
When you say "throttle sources'" here, you may want to link over to where we explain what a throttle source is (https://iqmotion.readthedocs.io/en/feature-redundant_throttle/manual/manual_throttle.html#throttle-sources)